app/_data/tables/support/gateway/versions/33.yml

```
@@ -0,0 +1,93 @@
+{% include_relative_once _data/tables/support/gateway/packages.yml %}
+{% include_relative_once _data/tables/support/gateway/third-party.yml %}
+{% include_relative_once _data/tables/support/gateway/browsers.yml %}
+
+lts: false
+distributions:
+  - <<: *alpine
+    docker: true
+  - <<: *amazonlinux2
+    docker: true
+    arm: true
+  - <<: *amazonlinux2023
+    docker: true
+    arm: true
+  - *centos7
+  - *debian10
+  - <<: *debian11
+    docker: true
+  - *rhel7
+  - <<: *rhel8
+    docker: true
+    fips: true
+  - <<: *ubuntu1804
+    arm: false
+    docker: false
+  - <<: *ubuntu2004
+    arm: false
+    docker: false
+    fips: true
+  - <<: *ubuntu2204
+    arm: true
+    docker: true
+    fips: true
+
+third-party:
+  datastore:
+    - *postgres
```

```suggestion
    - <<: *postgres
      versions:
        - Amazon Aurora
```
Adding Amazon Aurora per Tom's request. The main doc for it is [here](https://github.com/Kong/docs.konghq.com/pull/5503), but I'd rather make sure to update the support tables all in one place.